docs/environments/archive-environment.md

```diff
@@ -0,0 +1,22 @@
+---
+draft: true
+title: Archive an Environment
+intro: Some environments have a specific time frame that they will be used, proof of concepts and testing can be examples of such use cases; after their goal has been achieved, you might want to claim back those resources while ensuring a gradual transition of the users dependent on the environment. Archiving unused environments avoids you being charged by actions on old projects while keeping your environment one click away from being available to your users again.
```

Comment:
1. I believe the context should be set in a lighter, non-specific way, and only then mention examples to the reader.
Regardless of an environment `having a specific time frame` or not, a user might decide do archive it, so the docs should not actually say `when` it should be done but, instead, we can give examples of scenarios that might motivate the decision of performing this operation.

A possible, lighter, introduction could be something along the lines of:
```markdown
intro: You can archive an environment to make it read-only for all users as a way to indicate that 
it's no longer actively maintained. When an environment is archived all its data is preserved, 
including the environment settings, team memberships and environment resources. 
Archived environments can be unarchived at any time.

> Note #1: Archiving an environment does not remove its cloud resources from the 
corresponding cloud providers where they were provisioned.
For cost savings, if an environment has cloud resources that are no longer needed, 
it's recommended that you remove the resources using Devopness before 
archiving an environment.

> Note #2: Archived environments can only be accessible, in read-only mode, by the user 
who owns the project to which the environment belongs to. If other users need access to 
an archived environment, please unarchive it first. For more information, 
see <link to post `unarchive-environment`>
```

2. Sentences like `you might want to claim back those resources` might be a bit misleading, as the cloud resources remain available to the user on their corresponding CSPs, and remain belonging to the user all the time, so I can't see a need of `claiming back those resources`.

Reply:
Agreed, post updated